docs/usage.md

```diff
@@ -987,6 +991,22 @@ Specify to skip over regions of high depth by discarding alignments overlapping
 
 Specify ploidy of sample in FreeBayes. Default is diploid. Default: 2
 
+#### `--pileupcaller_bedfile`
+
+Specify a SNP panel in the form of a bed file of sites at which to generate pileup for pileupCaller.
+
+#### `--pileupcaller_snpfile`
+
+Specify a SNP panel in [EIGENSTRAT](https://github.com/DReichLab/EIG/tree/master/CONVERTF) format, pileupCaller will call these sites.
+
+#### `--pileupcaller_majority_call`
```

Comment:
I guess if random diploid is default, it would make more sense to put that first for readability (sorry for not picking up first time aroud). Could you update (and update the ToC).